examples/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 if(WITH_OTLP)
   add_subdirectory(otlp)
+  add_subdirectory(grpc)
```

Comment:
We can use a separate cmake option too - but I felt that it's better to build this along with otlp as both have dependency on grpc library.